### ISSUE TITLE
FB.ui(...

### ISSUE THREAD

**Issue by junit38** [Thuzi/facebook-node-sdk#79]
Hi, does there is a way to create a Facebook share dialog with your SDK ?

Here is the link:
https://developers.facebook.com/docs/sharing/reference/share-dialog

Thanks.

**Comment by vnodecg** [comment]
I don't think it does. Its a backend SDK. If you want to create share dialog consider including facebook's jssdk in ur page
